### ISSUE TITLE
Faster Whisper large-v2 model is repeating the segments

### ISSUE THREAD

**Issue by Talhazeb** [guillaumekln/faster-whisper#424]
Hi i am currently using faster whisper large-v2 model with german language, and it is repeating the same text in loop. I am not able to find the issue in the faster whisper, but the same file whith openai/whisper does not produces the same segements in loop. here is my code for the transcriptioin using faster whisper large-v2.
`
from faster_whisper import WhisperModel
import time

model_whisper=WhisperModel("large-v2", device="cuda", compute_type="float32",device_index=[0])

segments,info=model_whisper.transcribe("../../test-audio/dog.MP3",beam_size=5,language='de',vad_filter=True)
for segment in segments:
    print(segment.text)
`
ouput produces the same segments:
Guten Tag und herzlich Willkcmen bei
Herzlich Willkcmen bei der Deutschen
Herzlich Willkcmen bei der Deutschen
Herzlich Willkcmen bei der Deutschen
Herzlich Willkcmen bei der Deutschen
Ich rufe aus "private data".
Ich rufe aus "private data".
der Deutschen t&dien.
Ich interessiere mich für den Glasfaserausbau.
Ich interessiere mich für den Glasfaserausbau.
Ich würde gerne einen Beratungstermin vereinbaren.
Ich würde gerne einen Beratungstermin vereinbaren.
Gibt mir eine ganz kurze Postleitzahl.
Gibt mir eine ganz kurze Postleitzahl.
Beratung können Sie nur online beantragen.
Beratung können Sie nur online beantragen.

**Comment by Purfview** [comment]
Could you share the audio sample to reproduce the issue?

**Comment by Purfview** [comment]
purfview [@] protonmail [.] com

**Comment by Purfview** [comment]
I didn't got any repeats, my settings used:
`--device=cpu --language=de --model=large-v2 --compute_type=float32 --beam_size=5`

Make sure you are using the latest `0.7.1` version.

**Comment by guillaumekln** [comment]
@Talhazeb Did you try with the latest version?

**Comment by Talhazeb** [comment]
@guillaumekln Yes with latest version

**Comment by Purfview** [comment]
Do you get repeats with settings I used?
Only difference from yours were `device=cpu` and `vad_filter=False`.

**Comment by zyokia** [comment]
+1 here. My video  last about 1 hour.
with --model large-v2

**Comment by guillaumekln** [comment]
Please share the input audio file if possible.

**Comment by Talhazeb** [comment]
@Purfview I need the vad_filter since disabling it creates problem for other audio files.
@guillaumekln can you kindly share your mail, I can send you on that. Thanks

**Comment by guillaumekln** [comment]
guillaume [.] klein [@] systrangroup [.] com

**Comment by Talhazeb** [comment]
@guillaumekln sent

**Comment by Talhazeb** [comment]
@guillaumekln Thanks a lot for checking it out and letting me know. I will check and let you know.

**Comment by makoto-toyouke** [comment]
I have also likely faced the same issue. Adjusting the `min_silence_duration_ms` parameter causes the phenomenon of repeating the same segment to occur in other places. It would be very time-consuming if we have to repeatedly test each audio file to find out which value to set to prevent such occurrences. I would like to automate this part as well.

**Comment by Haarris** [comment]
@makoto-toyouke 
That's true. its a troublesome process as noise vary from 1 audio to other. Any one got any solution?
